rancher-ui-2.10.yaml

```diff
@@ -0,0 +1,98 @@
+package:
+  name: rancher-ui-2.10
+  version: "2.10.4"
+  epoch: 0
+  description: Rancher UI
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - git
+      - npm
+      - nvm
+      - perl
+      - py3-supported-build-base-dev
```

Comment:
We can omit this, this will install all versions of Python + build base. We just need 3.10